aws/resource_aws_codepipeline.go

```diff
@@ -434,6 +442,10 @@ func flattenAwsCodePipelineStageActions(actions []*codepipeline.ActionDeclaratio
 			values["region"] = *action.Region
 		}
 
+		if action.Namespace != nil {
+			values["namespace"] = *action.Namespace
```

Comment:
can you change this to `values["namespace"] = aws.StringValue(action.Namespace)`

Reply:
Updated